File: clangua/question3.c
#include <stdio.h>

int main() {
    
    int side1,side2,side3;
    printf("Enter a side1: ");
    scanf("%d",&side1);
    
    printf("Enter a side2: ");
    scanf("%d",&side2);
    
    printf("Enter a side3: ");
    scanf("%d",&side3);
    
    if(side1==side2 && side2==side3){
    printf("The triangle is a equilateral triangle.");
}else if(side1==side2 || side2==side3 || side3==side1){
    printf("The triangle is a isosceles triangle.");
}else{
    printf("the triangle is a scalene triangle.");
}

return 0 ;
}

File: js/question3.js
var side1=parseInt(prompt("Enter a side1: "))
var side2=parseInt(prompt("Enter a side2: "))
var side3=parseInt(prompt("Enter a side3: "))

if(side1===side2 && side2===side3){
    console.log("The triangle is a equilateral triangle.")
}else if(side1==side2 || side2==side3 || side3==side1){
    console.log("The triangle is a isosceles triangle.")
}else{
    console.log("the triangle is a scalene triangle.")
}